streams/de.m3u

```diff
@@ -43,6 +43,9 @@ https://derste247liveint.akamaized.net/hls/live/662735/daserste_int/master.m3u8
 https://mcdn.daserste.de/daserste/de/master.m3u8
 #EXTINF:-1 tvg-id="DaVinci.de" status="timeout",Da Vinci
 http://sc.id-tv.kz/DaVinci_38_39.m3u8
+#EXTINF:-1 tvg-id="DaVinciAsia.de" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",Da Vinci Asia (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
+https://livecdn.fptplay.net/hda2/davincihd_vhls.smil/chunklist.m3u8
```

Comment:
Put all of those into the vn_fptplay.m3u file, and keep the original notations.

Reply:
Will do that soon.

Reply:
> Put all of those into the vn_fptplay.m3u file, and keep the original notations.

You might also want move other fptplay URLs to file you have mentioned. I'm only move fptplay URLs that is part of my pull request.